screening/x-road.json

```diff
@@ -0,0 +1,111 @@
+{
+  "name": "X-Road",
+  "ownership": {
+    "isOwnershipExplicit": "Yes",
+    "copyrightURL": "https://github.com/nordic-institute/X-Road-development"
+  },
+  "dependencies": {
+    "mandatoryDepsCreateMoreRestrictions": "No",
+    "isSoftwarePltIndependent": "",
+    "pltIndependenceDesc": ""
+  },
+  "documentation": {
+    "isDocumentationAvailable": "Yes",
+    "documentationURL": [
+      "https://x-road.global/resources"
+    ]
+  },
+  "NonPII": {
+    "collectsNonPII": "No",
+    "nonPIIAccessMechanism": ""
+  },
+  "privacy": {
+    "isPrivacyCompliant": "Yes",
+    "privacyComplianceList": [
+      "GDPR"
+    ],
+    "isLocIntlAdherent": "Yes",
+    "locIntlComplianceList": [
+      ""
+    ]
+  },
+  "standards": {
+    "supportStandards": "Yes",
+    "standardsList": [
+      "RFC6960, RFC3161, ETSI TS 102 918"
```

Comment:
Shouldn't this be an array? Currently it is simply a string.

Reply:
Changed items in "standardsList" from string to array.